test/e2e/specs/blocks/shared/block-smoke-testing.ts

```diff
@@ -54,7 +54,7 @@ export function createBlockTests( specName: string, blockFlows: BlockFlow[] ): v
 					const blockHandle = await editorPage.addBlockFromSidebar(
 						blockFlow.blockSidebarName,
 						blockFlow.blockEditorSelector,
-						{ noSearch: true }
+						{ noSearch: false }
```

Comment:
**question:blocking** - this option is set to 'true' for performance and stability reasons, is it necessary to flip this boolean?

Reply:
Looks like the fix from @dpasque also fixed this!